cmake/modules/SwiftSupport.cmake

```diff
@@ -61,6 +61,32 @@ function(get_swift_host_os result_var_name)
   endif()
 endfunction()
 
+# Returns the module triple in a variable. This is cached for future use.
+#
+# Usage:
+#   get_swift_module_triple(result_var_name)
+#
+# Sets ${result_var_name} with the module triple derived from the Swift compiler.
+# This is cached in the Swift_MODULE_TRIPLE variable.
+function(get_swift_module_triple result_var_name)
```

Comment:
Rather than making it a function, why don't we set it up early on in the build so that the value is just available for use?

Reply:
Done, thanks!